### PR TITLE
[WIP] additional log output if check fails

### PR DESCRIPTION
**Pull request by coreydaley**
Dump additional application log output in reference to https://github.com/openshift/origin/issues/16229

### CODE REVIEW

**Comment by coreydaley** [comment]
@bparees ptal

**Comment by kargakis** [comment]
@stevekuznetsov do we need to enhance the wip regexp? It's silly.

**Comment by stevekuznetsov** [comment]
it got shot down upstream ... but we could, i think i originally had `.WIP.`

FWIW @coreydaley is the first person I've ever seen use parens :)

**Comment by coreydaley** [comment]
@stevekuznetsov I've used parens several times for that ;) 
Tell me what you want me to use and I will (when i remember)

**Comment by stevekuznetsov** [comment]
The bot recognizes `WIP` and `[WIP]` as prefixes but you can just `/hold` and `/hold cancel` and not need to edit your title

**Comment by coreydaley** [comment]
@stevekuznetsov I WIP them so that @bparees et al will not spend time reviewing it until it's ready.  But maybe I can retrain him to respect the new syntax

File: test/extended/image_ecosystem/s2i_perl.go [hunk 1]
```diff
@@ -58,7 +58,10 @@ var _ = g.Describe("[image_ecosystem][perl][Slow] hot deploy for openshift perl
 				_, err := exutil.WaitForPods(oc.KubeClient().Core().Pods(oc.Namespace()), dcLabel, exutil.CheckPodIsRunningFn, 1, 4*time.Minute)
 				o.ExpectWithOffset(1, err).NotTo(o.HaveOccurred())
 
-				result, err := CheckPageContains(oc, dcName, "", pageCountFn(i))
+				result, err := CheckPageContains(oc, dcName, "", pageCountFn(i), i)
+				if err != nil || !result {
+					exutil.DumpApplicationPodLogs(dcName, oc)
```
Comment (bparees) [inline test/extended/image_ecosystem/s2i_perl.go]: fyi i've added log dumping for all pods in the namespace on any test failure, to all our extended tests, so i think you can remove these.

**Comment by bparees** [comment]
@coreydaley bump.

**Pull request closed without merging**